admin/spam/serializers.py

```diff
@@ -0,0 +1,48 @@
+from website.project.model import Comment, User
+from website.profile.utils import serialize_user
+from modularodm import Q
+
+
+def serialize_comment(comment, full=False):
+    comment = {
+        'id': comment._id,
+        'author': serialize_user(comment.user, full=full),
+        'date_created': comment.date_created,
+        'date_modified': comment.date_modified,
+        'content': comment.content,
+        'has_children': bool(getattr(comment, 'commented', [])),
+        'modified': comment.modified,
+        'is_deleted': comment.is_deleted,
+        'reports': serialize_reports(comment.reports),
+        'node': comment.node,
+    }
+    comment.update(
+        category=comment['reports'][0]['category'],
+    )
+    return comment
+
+
+def retrieve_comment(id, full_user=False):
+    query = Q("_id", 'eq', id)
```

Comment:
You don't need the intermediate variable here or on line 32. It appears to be common practice to just do:
```python
comment = Comment.find_one(Q('_id', 'eq', id)
```
Since you're doing a primary key lookup you could even just do:
```python
comment = Comment.load(id)
```
Not sure of the differences between the error states of those two methods if an item with that id isn't found.